### PR TITLE
Add stripe import script

### PR DESCRIPTION
**Pull request by relud**

### CODE REVIEW

**Review by acmiyaguchi — COMMENTED** [review]
Is there a bug for context?

One suggestion is to dump out a schema from the initial testing of this script and use it to initialize the table in the script. It would help document the output and force the job to fail early if it unexpectedly changes.

File: bigquery_etl/stripe/import.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    if value.islower():
+        result = getattr(stripe, value.capitalize())
+    else:
+        result = getattr(stripe, value)
+    if not issubclass(result, ListableAPIResource):
+        raise ValueError("resource not listable")
+    return result
+
+
+parser = ArgumentParser(description=__doc__)
```
[the remark this answers is not in the document]
Comment (relud) [reply bigquery_etl/stripe/import.py]: moved to click/bqetl in 95d1002

**Comment by relud** [comment]
> Is there a bug for context?

there is not, so I'll give some here:

Stripe handles payments for mozilla vpn and other products, and provides an API and python SDK for reading their data. This allows us to [list top-level resources](https://stripe.com/docs/api/pagination), and the last 30-days of [events](https://stripe.com/docs/api/events).

This script imports stripe information into BigQuery so that we can make an executive dashboard using standard tooling. In order to create a full picture with an incomplete history of events, we need to import both events and the individual resources they impact.

> One suggestion is to dump out a schema from the initial testing of this script and use it to initialize the table in the script. It would help document the output and force the job to fail early if it unexpectedly changes.

I agree that this is important, but adding the schemas is going to take some time, and I would rather include it as a follow-up PR so that it doesn't block scheduling this in airflow rather than running it manually.

**Review by acmiyaguchi — APPROVED** [review]
r+

I still think it would be useful to commit the result of `bq show --format json {dataset}.{table} | jq '.schema'` into this module for documentation purposes, even if it isn't going to be used.

**Comment by relud** [comment]
> r+
> 
> I still think it would be useful to commit the result of `bq show --format json {dataset}.{table} | jq '.schema'` into this module for documentation purposes, even if it isn't going to be used.

mm, yeah, I can definitely do that.

**Pull request merged**